### ISSUE TITLE
Cannot import images successfully

### ISSUE THREAD

**Issue by math-feng** [mgmeyers/obsidian-zotero-integration#333]
When I import the image for the second time the following happens！
![image](https://github.com/mgmeyers/obsidian-zotero-integration/assets/78469397/60dbecb9-df85-4b38-ac61-336daceb5918)
My import template uses `persist` and I'm not sure if that has anything to do with it；When I restarted Obsidian, the image was successfully displayed instead

**Comment by adjiap** [comment]
Hey @math-feng, I just found a really weird workaround. I guess this is a bug somewhere, though I'm not sure whether it's BetterBibText, or it's this plugin, but did you try to run the Command Palette, "Reload App Without Saving"?

Weirdly, once it restarted, it showed all the images.

**Comment by math-feng** [comment]
> Hey @math-feng, I just found a really weird workaround. I guess this is a bug somewhere, though I'm not sure whether it's BetterBibText, or it's this plugin, but did you try to run the Command Palette, "Reload App Without Saving"?
> 
> Weirdly, once it restarted, it showed all the images.

Indeed, as soon as I restarted Obsidian, all the images showed up properly!

**Comment by CurryMultiSci** [comment]
> > Hey @math-feng, I just found a really weird workaround. I guess this is a bug somewhere, though I'm not sure whether it's BetterBibText, or it's this plugin, but did you try to run the Command Palette, "Reload App Without Saving"?
> > Weirdly, once it restarted, it showed all the images.
> 
> Indeed, as soon as I restarted Obsidian, all the images showed up properly!

Thank you! It also works for my case! So strange!
